### PR TITLE
Fix a couple of bugs.

### PR DESCRIPTION
**Pull request by rlepigre**
CC @Janno.

### CODE REVIEW

File: src/coq_elpi_builtins.ml [hunk 1]
```diff
@@ -1855,7 +1855,13 @@ Supported attributes:
...
   DocAbove);
 
-  Coq_elpi_builtins_synterp.current_path;
+  MLCode(Pred("coq.env.current-path",
+    Out(list B.string, "Path",
+    Read(unit_ctx, "lists the current module path")),
+  (fun _ ~depth _ _ state -> !: (mp2path (Global.current_modpath ())))),
+  DocAbove);
+
+  (* FIXME should be different from the Synterp version. *)
```
Comment (gares) [inline src/coq_elpi_builtins.ml:1864]: ?

**Review by gares — APPROVED** [review]
I'd merge, but what is this comment?
Do you want to add an assert?

**Comment by gares** [comment]
Thanks!

File: src/coq_elpi_builtins.ml (hunk 1 above)
Comment (rlepigre) [reply src/coq_elpi_builtins.ml:1864]: Basically, the following builtin has the same problem, but it cannot be fixed as the information is not available at the `interp` phase. There are several options: removing the comment and ignoring this, leaving the comment to remind us that there are problems with this primitive, removing the primitive (only at `interp`).

Comment (gares) [reply src/coq_elpi_builtins.ml:1864]: I think an issue and a pointer to that issue in the doc of the API would be good

**Review by gares — APPROVED** [review]

**Pull request merged**